packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -242,12 +269,12 @@ class Radio extends React.Component<Props> {
     };
 
     render(): React.Node {
-        let choices = this.props.choices;
-        let choiceStates;
+        const choices: $ReadOnlyArray<PerseusRadioChoice> = this.props.choices;
+        let choiceStates: $ReadOnlyArray<ChoiceState>;
         if (this.props.static) {
-            choiceStates = _.map(choices, (val) => ({
-                selected: val.correct,
-                crossedOut: val.crossedOut,
+            choiceStates = choices.map((choice) => ({
+                selected: !!choice.correct,
+                crossedOut: !!choice.crossedOut,
```

Comment:
**suggestion (non-blocking):** I don't understand why we'd be using `!!choice.crossedOut` here.  It seems like this should always be `false` since `updateChoices()` modifies `choiceStates`.  I would try setting this to `crossedOut: false` and see if any tests break.

Reply:
Looks like it's fine with changing this to false. I wasn't quite sure what static meant, so I looked it up in types.js. It says, "When determining if a widget is static, we verify that the widget is not an exercise question by verifying that it has no problem number." So it would make sense in the case of a static widget without any exercise questions that crossedOut would be false. Not sure why a radio widget is used if it is not an exercise question though. Is this the correct reasoning why it can be false?

Reply:
That reasoning makes sense to me.  One way that we could try to verify if `choice.crossedOut` here is ever non-falsy is to throw an exception if it's non-falsy and then run the tests to see if anything blows up.